src/Stratis.Bitcoin/P2P/PeerConnectorDiscovery.cs

```diff
@@ -13,6 +13,8 @@ namespace Stratis.Bitcoin.P2P
     /// </summary>
     public sealed class PeerConnectorDiscovery : PeerConnector
     {
+        private const int MaximumPeerSelectionAttempts = 50;
```

Comment:
This actually fixes issue #955 . Can you change this to 5?

Reply:
Done.